mobify/astro-scaffold
# Add an adaptive bundle for Velo

Linked PRs: Velo PR for adding Mobify tag (dependency) https://github.com/mobify/velo/pull/14

The cloud project can be found here: https://cloud.mobify.com/projects/velo/publishing/production/
## Changes
- Add a simple adaptive bundle for Velo. All it does is remove the unnecessary header/menu.
## How to test-drive this PR
- Clone Velo repo. Switch to `add-adaptive` branch. Run Velo locally with `npm i && harp server`
- Serve the adaptive bundle locally with `cd adaptive && grunt preview`
- In `app/app-config/baseConfig.js`, change `baseURL` to your locally running Velo (should be `http://localhost:9000`) and make sure `previewBundle` is your locally running bundle (should be `https://localhost:8443/adaptive.js`)
- FINALLY run the scaffold and click PREVIEW
- Observe the header is gone. lol.
## TODOS:
- [ ] Publish bundle!
- [ ] Deploy Velo
- [ ] Generator: **Make sure the adaptive folder is ignored!!**
- [ ] Update `.gitignore` to ignore files generally needed to be ignored inside `adaptive/`
